### ISSUE TITLE
Plugin: Provide a way to configure the update interval for top

### ISSUE THREAD

**Issue by khurshid-alam** [kupferlauncher/kupfer#82]
### Actual Behaviour

By default it updates every minute. I may not want to do that frequently specially when my system is idle. Top itself provides to configure to update interval. It would be nice if we can configure from plugin preference.

### Environment

Kupfer Version: 317

**Comment by bluss** [comment]
Good point. The default is actually every 5 seconds. (The 60 figure is the timeout it uses before the AsyncCommand gives up on its command, it's just a timeout).

This plugin doesn't feel like it fits very well into Kupfer to me. It has no quick-access factor.

**Comment by khurshid-alam** [comment]
I only use to kill a certain process when it hangs (and system-monitor stops responding). But 5 sec default is simply too frequent in my opinion.

**Comment by bluss** [comment]
Sure, it should have a config

**Comment by bluss** [comment]
Ok, looking closer, it only triggers refresh when updating items. This means that in a totally idle state, it never updates. (Not until the kupfer global rescan cycle comes around, which is fixed to once hour).

This means that if Kupfer is not being actively used, it's not polling top in the background.

**Comment by KarolBedkowski** [comment]
In v323 rescan mechanic was changed, so this may relevant anymore.
Fill free to reopen if necessary.
